softhsm/Dockerfile

```diff
@@ -1,27 +1,33 @@
-from alpine:latest as build
-workdir /softhsm
+from openjdk:11-jre-slim as build
+workdir /app
 
 # Build softHSM source
 arg version=2.5.0
-run apk --update add opensc build-base wget sqlite-dev openssl-dev
+
+run apt-get update && \
+    apt-get install -y \
+    opensc build-essential wget libsqlite3-dev libssl-dev \
+    cmake libseccomp-dev libengine-gost-openssl1.1
```

Comment:
`libengine-gost-openssl1.1` seems to be missing from Debian Stretch, this can be fixed by adding `--disable-gost` to `./configure`